### PR TITLE
[FbCDN.net] Remove targets that are gone from DNS

### PR DESCRIPTION
**Pull request by Bisaloo**
Again, it's fine if you need to take some time to review this ruleset.

### CODE REVIEW

**Comment by J0WI** [comment]
@jeremyn could you please have an additional look here?

**Comment by jeremyn** [comment]
I agree with all the deletions and changing the `securecookie` to `.+`/`.+` to match the preferred convention in [`CONTRIBUTING.md`](https://github.com/EFForg/https-everywhere/blob/a50b6f21347039b92a12dcebc3d4465fad7167a5/CONTRIBUTING.md#regex-conventions).

The two new `test` URLs return a `404` at their HTTPS equivalents:

* https://external-cdg2-1.xx.fbcdn.net/static_map.php?v=29&#x26;osm_provider=2
* https://static.xx.fbcdn.net/images/emoji.php

Normally we want URLs with `2XX`/`3XX` responses. Can we use other URLs?

**Comment by Bisaloo** [comment]
Done. It's not so easy to find good test urls without leaking any private info with this kind of domain.

File: src/chrome/content/rules/FbCDN.net.xml [hunk 1]
```diff
@@ -73,8 +55,11 @@
 
 	<!--	Direct rewrites:
 				-->
-	<target host=   "*.xx.fbcdn.net" />
+	<target host="*.xx.fbcdn.net" />
 
+		<test url="http://external-cdg2-1.xx.fbcdn.net/map_tile.php?v=29&#x26;osm_provider=2&#x26;x=0&#x26;y=0&#x26;z=0" />
+		<test url="http://external-lht6-1.xx.fbcdn.net/map_tile.php?v=29&#x26;osm_provider=2&#x26;x=0&#x26;y=0&#x26;z=0" />
```
Comment (J0WI) [inline src/chrome/content/rules/FbCDN.net.xml]: I still gets 404s here.

Comment (Bisaloo) [inline src/chrome/content/rules/FbCDN.net.xml]: @J0WI, did you change `&#26;` to `&`?

**Pull request merged**